src/select.jsx

```diff
@@ -369,7 +369,8 @@ var classBase = React.createClass({
           tabIndex={-1}
           aria-hidden={true} >
             {React.Children.map(this.props.children, function (child, index) {
-              return <option key={index} value={child.props.value}>{child.props.children}</option>
+              var value = child.props.value || child.props.children;
+              return <option key={index} value={child.props.value}>{value}</option>
```

Comment:
Nevermind. You were originally correct with `>{child.props.value}<`. Sorry for the confusion, I thought this was the rendered children not the hidden select.